stdlib/release/src/release/trigger_workflows.bal

```diff
@@ -0,0 +1,125 @@
+import ballerina/config;
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerina/runtime;
+
+public function main() {
+    http:Client httpClient = new (API_PATH);
+    string accessToken = config:getAsString(ACCESS_TOKEN_ENV);
+    string accessTokenHeaderValue = "Bearer " + accessToken;
+    http:Request request = createRequest(accessTokenHeaderValue);
```

Comment:
Shall we move this request creation into another function and call it whenever we want the request? I see that the request is being used at #L55 and #L80. 
Also, we can use separate requests for each call rather than reusing the same request if you do so.